### DISCUSSION TITLE
Real toric varieties

### DISCUSSION THREAD

**Discussion by MVallee1998** [oscar-system/Oscar.jl#2055] [Ideas]
Dear developers,

The current work on this Julia package is auspicious and has picked my interest.
I want to suggest some new functionalities.
To any complex toric variety $X(\Sigma)$, one can associate a real toric variety $X^{\mathbb{R}}(\Sigma^{\mathbb{R}})$ which is its real locus and whose fan $\Sigma^{\mathbb{R}}$ is the mod 2 reduction of $\Sigma$.
Just as in the complex case, the cohomology of $X^{\mathbb{R}}$ can be computed from the mod 2 fan and is a quotient ring in $\mathbb{Z}/2\mathbb{Z}$ by some ideal coming from the mod 2 fan. An explicit example is computed by hand [here](https://link.springer.com/10.1134/S0081543822020109).
The $\mathbb{Z}/4\mathbb{Z}$-homology can also be computed easily from the mod 2 fan and the simplicial $\mathbb{Z}/2\mathbb{Z}$-homology of the simplicial complex underlying the mod 2 fan, the formula is provided [here](https://www.degruyter.com/document/doi/10.1515/forum-2016-0025/html).
Finally, there is a big question in toric topology called the 'lifting conjecture': given a complete nonsingular mod 2 fan $\Sigma^{\mathbb{R}}$, can we find a complete nonsingular (integral) fan $\Sigma$, called a lift, whose mod 2 reduction is $\Sigma^{\mathbb{R}}$? 
No efficient algorithm for finding such a lift exist.

Best regards,
Mathieu Vallée

**Comment by fingolfin** [comment]
Hi @MVallee1998, just to clarify: are you proposing to work on this? Or are you requesting that someone else should work on it?

**Reply by MVallee1998** [reply]
Hi @fingolfin ! I have never taken part in collaborating on such a project and am currently focused on my Ph.D.
Moreover, as this project is being developed quite fast, I am afraid I will not be able to keep a reasonable pace and follow all updates and pulls, which will maybe lead my contribution to be pointless (?)
Nevertheless, I can give it a try and implement these changes since I've already done it in Python before.
Best
